### PR TITLE
New package: skb-0.4

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/skb/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+makedepends="libX11-devel"
+short_desc="Simple keyboard layout indicator"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="GPL-2"
+homepage="http://plhk.ru/"
+distfiles="http://plhk.ru/static/skb/skb-${version}.tar.gz"
+checksum=635d1108ce9d86ac011f0871d1af6e3debb64700ee4682ecb764da0b629ffa64
+
+do_build() {
+	make CC=$CC INCS="-I." LIBS="-lX11" ${makejobs}
```
Comment (xtraeme) [inline srcpkgs/skb/template]: This is ignoring our CFLAGS/LDFLAGS.

**Comment by Gottox** [comment]
any news?

**Comment by diogoleal** [comment]
By the end of the week I solve this pending.

**Comment by Vaelatern** [comment]
Bump.

**Comment by Vaelatern** [comment]
Almost 2 months since this PR was opened, any luck?

**Comment by diogoleal** [comment]
Sorry for the delay in returning to work in this PR.

**Comment by Vaelatern** [comment]
It still seems to ignore our CFLAGS and LDFLAGS

**Pull request closed without merging**